aws/modules_common.tf

```diff
@@ -15,12 +15,11 @@ module "codebuild_cd-filecoin-fluent-bit" {
   ]
 }
 
-module "codebuild_ci_cid-checker" {
+module "codebuild_cd_cid-checker_mainnet" {
```

Comment:
looks like you made a push build job for env
and delete a build module for it.

Reply:
fixed